integration_tests/sdk/shared/flow_helpers.py

```diff
@@ -82,6 +88,23 @@ def publish_flow_test(
     )
     print("Workflow registration succeeded. Workflow ID %s. Name: %s" % (flow.id(), name))
 
+    engine_type = type_from_engine_name(client, engine)
+    if engine_type == ServiceType.AIRFLOW:
+        # Copy over the Airflow DAG file to the Airflow DAGs folder
+        dag_path = f"{name}_airflow.py"
+        assert os.path.exists(
+            dag_path
+        ), f"The expected Airflow DAG file was not found at {dag_path}"
+
+        folder_path = os.path.expanduser(AIRFLOW_DAGS_FOLDER)
+        shutil.move(dag_path, folder_path)
+
+        print("Sleeping for 20s to wait for Airflow scheduler to pick up new DAG file")
+        time.sleep(20)
```

Comment:
Is this sleeping guaranteed to work? What happens if the scheduler doesn't pick it up in time? Does trigger fail or something.

Reply:
We configure the scheduler to check for new DAGs every 15 seconds so this should be ok. We can bump it to 30s to be safe. Yes, trigger will fail if the scheduler doesn't pick up the new DAG.

Reply:
Configured here https://github.com/aqueducthq/aqueduct/pull/1204/files#diff-fd66f15b393a531842dfb747a1dd0c1cdc7ddbd210a4daafbda5ac23a3021d76R15